_docs/pages/instrumentation/common-use-cases.md

```diff
@@ -253,3 +253,69 @@ if request.get('debug'):
         tags={tags.SAMPLING_PRIORITY: 1}
     )
 ```
+
+### Tracing Messaging Scenarios
+
+There are two messaging styles that should be handled, Message Queues and Publish/Subscribe (Topics).
+
+The main distinction between these styles is that a message written to a queue can only be consumed by **at most** one consumer, whereas a message published on a topic can be consumed by zero or more subscribers. The other point to note is that although a message can be sent/published by a producer, that does not guarantee that it will be consumed - messaging systems are asynchronous and therefore the producer has no visibility of when or if the message was delivered.
```

Comment:
I think in a CAP world, if you choose AP system then "at most once" cannot be guaranteed, only "at least once" can be, which makes the distinction between queue and pub/sub even less clear. So I would drop that part, especially since your next paragraph says it doesn't really matter for tracing anyway.